### PR TITLE
[Bugfix-19848] Mention lockLoc allows arrow key movement

### PR DESCRIPTION
**Pull request by livecodealice**
Added a remark at the end of the lockLocation entry that it is still possible to move an object with the arrow keys while that property is set to true.

### CODE REVIEW

File: docs/notes/Bugfix-19848.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Clarified in the docs how objects can still move when lockLocation is true
```
Comment (livecodepanos) [inline docs/notes/Bugfix-19848.md]: @livecodesam The filename must change to `docs/notes/bugfix-19848.md`

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok cf690ef

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok cf690efd9b2c6e9d30d19e7ed66125ebfe14754b

**Comment by livecode-vulcan** [comment]
😎 test success cf690efd9b2c6e9d30d19e7ed66125ebfe14754b

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/766)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/175)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/175)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/175)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/175)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2292)
* try-community-universal-ios-iphoneos13.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos13.4/builds/26)
* try-community-universal-ios-iphonesimulator13.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator13.4/builds/26)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2321)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2308)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2307)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2342)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2282)

**Pull request merged**